app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -224,7 +174,7 @@ It's important that you give yourself time to write the best possible [personal
 
 Your personal statement can be up to 1000 words. 90% of successful candidates write 500 words or more.
 
-Your [teacher training adviser](#get-support-and-advice) can help you write your personal statement and will also give advice on: 
+If you have a [teacher training adviser](/help-and-support), they can help you write your personal statement and will also give advice on: 
 
 * your choice of course and training provider
```

Comment:
@laurahermionetennant92 your in the bullets can go in the line above - give advice on your:

Reply:
Um no cos of 'ways to research'

Reply:
and 'preparing for your interview'